generate/generate.go

```diff
@@ -137,6 +143,7 @@ func New() Generator {
 					Type: "mount",
 				},
 			},
+			Seccomp: &rspec.Seccomp{},
```

Comment:
Instead of adding an empty object here, the current pattern (since the recent #144) is to use an `init*` guard in each `Generator` method that requires the struct.